### PR TITLE
Repository

### PR DESCRIPTION
**Pull request by saarazari5**
added some new logic to retrofit

### CODE REVIEW

File: app/src/main/java/com/example/whatsapp_clone/Views/MainActivity.java [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 public class MainActivity extends AppCompatActivity {
 
-    public State state;
+    public State state = State.LOGIN;
```
Comment (oriKatzir) [inline app/src/main/java/com/example/whatsapp_clone/Views/MainActivity.java:21]: why is this outside of the onStart method of the main activity?

**Review by oriKatzir — COMMENTED** [review]
looks great!

File: app/src/main/java/com/example/whatsapp_clone/Views/MainActivity.java (hunk 1 above)
Comment (saarazari5) [reply app/src/main/java/com/example/whatsapp_clone/Views/MainActivity.java:21]: Because it's a class variable that the fragments need access too

**Comment by oriKatzir** [comment]
great

**Pull request merged**